pkg/config/default.go

```diff
@@ -0,0 +1,92 @@
+package config
+
+import (
+	"fmt"
+	"time"
+
+	prom "github.com/directxman12/k8s-prometheus-adapter/pkg/client"
+	pmodel "github.com/prometheus/common/model"
+)
+
+// DefaultConfig returns a configuration equivalent to the former
+// pre-advanced-config settings.  This means that "normal" series labels
+// will be of the form `<prefix>${.Resource}$`, cadvisor series will be
+// of the form `container_`, and have the label `pod_name`.  Any series ending
+// in total will be treated as a rate metric.
+func DefaultConfig(rateInterval time.Duration, labelPrefix string) *MetricsDiscoveryConfig {
```

Comment:
Any reason why this can't just be a default config as in a yaml file? It seems odd that we have this "default" but double-parameterizable config.

Reply:
The in-program one makes the existing flags still work, so it's an easier switch-over.  If we have no desire to do that, the default config can be moved to a YAML file.  It also provides a nice out-of-the-box experience.

Reply:
The way people break is the same once we remove the flags, so I'd say let's break immediately and use the YAML file config option.